charts/harvester-cloud-provider/harvester-cloud-provider/0.1.300+up0.1.3/values.yaml

```diff
@@ -0,0 +1,71 @@
+# Default values for harvester-cloud-provider.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicasCount: 2
+
+image:
+  repository: rancher/harvester-cloud-provider
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+clusterName: ""
+cloudConfigPath: "/var/lib/rancher/rke2/etc/config-files/cloud-provider-config"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
```

Comment:
neither of these fields seem to be used above? confused about why it is in the values.yaml...

Reply:
should this also be exposed in csi-driver?